src/main/resources/public/css/stylesheet.css

```diff
@@ -55,7 +45,35 @@ span {
     margin-right: 2em;
 }
 
-html, body {
-    height:100vh;
```

Comment:
?? I personally like the one line style lol

Reply:
that's the standards set by my linter.
![Screenshot_20190330_231515](https://user-images.githubusercontent.com/43054535/55284295-38200100-5363-11e9-86d2-c75e3d6838ed.png)

Reply:
Ok if we're doing this I need an issue created assigned to everyone to use this linter, the contributing page of the wiki update to explain that the user should be using this linter, and I need you to do a separate PR to lint all the files at once so we don't have linting changes from previous code creeping in whenever these files are modified for another purpose

Reply:
Done at https://github.com/amazin-team/Amazin-online-bookstore/wiki/Contributing